src/up.cmd.js

```diff
@@ -68,14 +68,15 @@ class UpCommand extends BuildCommand {
     this._watcher = watch(this.directory, {
       recursive: true,
     }, (eventType, filename) => {
-      if (filename.indexOf('src/') < 0 && filename !== HELIX_CONFIG) {
+      const file = filename.split(process.cwd()).join(''); // only consider paths starting from project root
```

Comment:
can't you use `this.directory` instead of `cwd`?
or better, `path.relative(this.directory, filename)`

I think the new watcher introduced by @trieloff now returns absolute paths. the previous one, returned relative ones.

Reply:
Good suggestion. Based on the original conditional, if the watcher now returns absolute paths, then without this fix, the check against `HELIX_CONFIG` also won't work.